.travis.yml

```diff
@@ -5,6 +5,32 @@ language: ruby
 cache: bundler
 
 rvm:
-  - ruby-head
-  - 2.2.2
-  - 2.3.1
\ No newline at end of file
+  - 2.4.1
+  - 2.3.4
+
+gemfile:
+  - Gemfile
+  - gemfiles/with_activerecord.gemfile
+  - gemfiles/with_mongoid.gemfile
+
+services:
```

Comment:
These should be under the appropriate matrix includes or it installs unnecessary services. NBD.